cob_hardware_config/cob4-4/config/diagnostics_analyzers.yaml

```diff
@@ -53,7 +53,12 @@ analyzers:
       base:
         type: diagnostic_aggregator/GenericAnalyzer
         path: Base
-        contains: 'base_controller'
+        contains: 'base'
```

Comment:
as can be seen in https://github.com/ipa320/cob_robots/pull/337/files#diff-84461ff221db0b022bddf71f74f6f90eR56, @ipa-bnm had to change the pattern for the base diagnostics...